0TA_Template_Sketch/OTA.h

```diff
@@ -1,12 +1,27 @@
+#ifdef ESP32
 #include <WiFi.h>
 #include <ESPmDNS.h>
+#else
+#include <ESP8266WiFi.h>
+#include <ESP8266mDNS.h>
+#endif
+
+
 #include <WiFiUdp.h>
 #include <ArduinoOTA.h>
 #include <credentials.h>
 
 const char* ssid = mySSID;
 const char* password = myPASSWORD;
 
+#ifdef ESP32_RTOS && ESP32
```

Comment:
This is not how you check for two defines. Notice the compiler warning: `warning: extra tokens at end of #ifdef directive`.
The correct way would be: `#if defined(ESP32_RTOS) && defined(ESP32)`.

Reply:
I have fixed that.